PowerPointLabs/PowerPointLabs/SyncLab/Views/SyncFormatPaneItem.xaml.cs

```diff
@@ -123,10 +131,54 @@ public String Text
             set
             {
                 textBlock.Text = value;
-                textBlock.ToolTip = value;
+                toolTipName.Text = value;
+            }
+        }
+
+        #endregion
+
+        #region Helper Functions
+
+        private void UpdateToolTipBody()
+        {
+            String toolTipBodyText = "";
```

Comment:
Don't think it really makes a difference since tooltips are likely to be short but maybe its better to use a StringBuilder instead? (same for line 158)

Reply:
Good idea!